packages/components/src/internal/app/utils.ts

```diff
@@ -220,11 +220,15 @@ export function isAssayEnabled(moduleContext?: ModuleContext): boolean {
 }
 
 export function isAssayQCEnabled(moduleContext?: ModuleContext): boolean {
-    return (
-        isAssayEnabled(moduleContext) &&
-        hasPremiumModule(moduleContext) &&
-        isFeatureEnabled(ProductFeature.AssayQC, moduleContext)
-    );
+    // NK: The product tiers which include Assay QC are not fully defined.
+    // For now (v22.11+), we're going to continue offering Assay QC features until
+    // we can fully define all desired product tiers.
+    // return (
+    //     isAssayEnabled(moduleContext) &&
+    //     hasPremiumModule(moduleContext) &&
+    //     isFeatureEnabled(ProductFeature.AssayQC, moduleContext)
+    // );
```

Comment:
It's truly not enabled if the premium module doesn't exist. Any risk that we're going to show functionality when that module is not present?  If so, we could just not check for the feature flag here.

Reply:
The way I see it is that this check did not exist at before a few days ago. So wherever this is now checked was previously just showing the UI. I considered doing what you're saying but it seems like less of a reversion of the behavior.

Reply:
Sure, but we never had a distribution that might surface assayQC functionality without actually providing that functionality, which we do now with the lims_starter distribution, which doesn't include premium. We're not deploying that to any clients, of course, so no big deal either way, just seem more "safe" to check for the existence of the module.